Source/SocketAckEmitter.swift

```diff
@@ -44,6 +44,10 @@ public final class SocketAckEmitter : NSObject {
         
         socket.emitAck(ackNum, with: items)
     }
+        
+    public func isRequired() -> Bool {
+        return ackNum != -1;
```

Comment:
This should probably be a computed property rather than a method.

```swift
var isRequired: Bool {
    return ackNum != -1
}
```

But to take a step back. I don't remember required acks being a thing. Were they recently added?